### PR TITLE
feat: Added image component to the CL

### PR DESCRIPTION
**Pull request by JantuDeb**
**Image Variants**

- Responsive Image
- Image thumbnails
- Image link

image
image

### CODE REVIEW

**Comment by hussiantabha** [comment]
It looks clean and good just one suggestion If you have the preview enabled I think there is no need to attach gif. This is just a suggestion and you looks good for merge too!

**Review by ultronjs — APPROVED** [review]
Images are responsive on resizing the window. Code also looks good. Go ahead and merge

File: components/main.css [hunk 1]
```diff
@@ -393,15 +394,15 @@ padding: 0 1rem;
...
 }
 
-.radius-m {
+.radius-md {
   border-radius: 8px;
 }
 
-.radius-l {
+.radius-lg {
   border-radius: 16px;
```
Comment (Raghav888) [inline components/main.css:406]: Use rem

File: documentation/styles.css [hunk 2]
```diff
@@ -4,111 +4,116 @@
...
+  overflow: hidden;
 }
 
-
-.sidebar{
-    width: 250px;
-    background-color: var(--fu-blue-lighter);
-    padding-left: 1rem;
+.sidebar {
+  flex-basis: 250px;
```
Comment (Raghav888) [inline documentation/styles.css:31]: use rem

**Review by Raghav888 — APPROVED** [review]

File: components/image.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+.img-fluid{
+    max-width: 100%;
+    height:auto;
+}
+
```
Comment (rkapoor10) [inline components/image.css:6]: you may create a utility class for responsive images. Just for a better structure.

File: documentation/styles.css [hunk 4]
```diff
@@ -4,111 +4,116 @@
...
-    color: #4B5563;
+.nav-list li {
+  height: 2rem;
 }
 
-.nav-list li a:hover{
-    color: #1C64F2;
+.nav-list li a {
+  text-decoration: none;
+  color: #4b5563;
```
Comment (rkapoor10) [inline documentation/styles.css:63]: use variables for colors

File: documentation/styles.css [hunk 5]
```diff
@@ -4,111 +4,116 @@
...
-    background-color: transparent;
-    border-top:2px solid var(--fu-orange);
-    border-left:2px solid var(--fu-orange);
-    border-right:2px solid var(--fu-orange);
-    padding: 0.5rem;
-    color: var(--fu-primary);
-    border-top-left-radius: 8px;
-    border-top-right-radius: 8px;
+.nav-list li a:hover {
+  color: #1c64f2;
```
Comment (rkapoor10) [inline documentation/styles.css:67]: same as above

**Review by rkapoor10 — APPROVED** [review]
The code looks great. Good for merge.

**Pull request merged**